examples/bert/README.md

```diff
@@ -220,7 +226,8 @@ training for a few epochs to finetune the model.
 
 ```shell
 python3 examples/bert/run_glue_finetuning.py \
-    --saved_model_input path/to/model/ \
```

Comment:
Can probably just leave this as is?

We don't really need to the output model from fine-tuning to be saved anywhere usually. Really what we would want is to save some predictions in the GLUE submission format (tracked here https://github.com/keras-team/keras-nlp/issues/115).

Reply:
Yea, I feel that users may find the need to reevaluate the model? If we do not save the model then people won't have access to the finetuned model except finetuning again.